assembl/lib/frontend_urls.py

```diff
@@ -237,7 +238,7 @@ def get_discussion_url(self, request=None, force_v1=False):
             from assembl.views import create_get_route
             get_route = create_get_route(request, self.discussion)
             if force_v1:
-                route = get_route('home')
+                route = self.get_frontend_url('oldDebate', slug=self.discussion.slug)
```

Comment:
probably a merge issue here: the correct one is `route = get_route('home')`

Reply:
I did a rebase so should be fine by now